server/src/main/java/org/apache/iotdb/db/metadata/rescon/DataNodeSchemaQuotaManager.java

```diff
@@ -40,6 +44,7 @@ public void updateRemain(long totalCount) {
   }
 
   public void checkMeasurementLevel(int acquireNumber) throws SchemaQuotaExceededException {
+    logger.info("checkMeasurementLevel {},{},{}", level.name(), limit, remain.get());
```

Comment:
It seems this log will be record each there's timeseries registration. Try reduce the log frequency.